### PR TITLE
Rename disable_cache to enable_cache; change default to disable it.

### PR DESCRIPTION
**Pull request by colohan**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -164,7 +164,7 @@ def mappings_from(nm):
     vc = hass.data[VANTAGE_CONTROLLER]
 
     await hass.async_add_executor_job(
-        functools.partial(vc.load_xml_db, config.get(CONF_DISABLE_CACHE, False))
+        functools.partial(vc.load_xml_db, config.get(CONF_ENABLE_CACHE, False))
```
Comment (gjbadros) [inline __init__.py]: Don't we need to !config.get(...) to invert the value now that we inverted the sense of the flag?  The arg to load_xml_db is disable_cache, still.

Comment (colohan) [reply __init__.py]: You are right.  Serves me right for not testing this code properly.  ;-(

Fixed.  Also, I patched this into a new client and tested it this time.  :-)

**Comment by gjbadros** [comment]
Cool... I have a look when at my desk.  Thx!

**Review by gjbadros — COMMENTED** [review]
LGTM, thx for the not(..)

**Pull request merged**